### PR TITLE
Reorganize the website as per @kathrynLA

### PR DESCRIPTION
**Pull request by masinter**
please review
fix as necessary

### CODE REVIEW

File: content/en/_index.md [hunk 1]
```diff
@@ -1,41 +1,52 @@
...
-
-## Who is involved?
-
-We are some of the original developers and users of the system 30 years ago, joined by newcomers interested in software history and preservation, along with members of the Lisp community. We work with [organizational partners](about/partners). And with you! See [Get Involved](about/getinvolved/) for ways to get involved.
-
-## Testimonials
-
-[What have we lost?](https://www.youtube.com/watch?v=7RNbIEJvjUA&t=841s)  A presentation from the 2020 Remote Chaos Experience conference highlights some of the interesting aspects of Interlisp.  The presentation synopsis states it this way:
-> We have ended up in a world where UNIX and Windows have taken over, and most people have never experienced anything else. Over the years, though, many other system designs have come and gone, and some of those systems have had neat ideas that were nevertheless not enough to achieve commercial success. We will take you on a tour of a variety of those systems, talking about what makes them special.
+## Welcome to Interlisp.org
```
Comment (stumbo) [inline content/en/_index.md]: Here's what the main page looks like with these changes.  I'm not a fan of the blue boxes.  Is this what we envision it looking like, or are these placeholders?  The boxes hyperlink to other locations in the document.

What's the hook that is going to keep someone that arrives at this site digging deeper?  We need some pictures or short videos that give a feel for what Medley is - maybe a couple pictures we cycle through with images of the desktop or short videos of a window being opened, pictures from an old conference.  Something that will appeal to readers.

image

Comment (stumbo) [reply content/en/_index.md]: I'm going to tackle this PR piece by piece.  With 75 files it's a bit of a beast.  I might suggest as we make progress on individual sections we break them into separate PRs and merge them.  That will, hopefully, keep the change management somewhat simplified and start moving new content to the website sooner.

**Comment by stumbo** [comment]
The left side pane, the table of contents, needs to be fixed up.  This is driven by the header each markdown page has in it.  Here's what it currently looks like:
image

@KathrynLA  Do you have a vision of how all the different pages should be ordered?  For instance, on our home page we have a link to a vocabulary page.  It isn't in the table of contents.  I'd like it to be there, I assume towards the bottom.  I'll fix it up if you can provide an outline of the ToC.

**Comment by KathrynLA** [comment]
@Bill
Attached please find my draft of an outline. It is very short and
preliminary. As I learn more it may grow and change.
--Kathryn

**Comment by stumbo** [comment]
I believe everything is ready to merge.  My test site, [stumbo.github.io](https://stumbo.github.io) reflects the current status of this pull request.  

I'll merge and deploy tomorrow if no new issues are identified.

**Comment by nbriggs** [comment]
In the left-hand column, on the vocabulary page (and any page with the same left-hand column), the search box is pre-filled with "Search this sit" instead of "Search this site".

**Comment by stumbo** [comment]
Nick,

What browser are you using?  I suspect the problem you're seeing is 
related to page size and the responsive design of Hugo.

Here's what I'm seeing.

  * Firefox on Ubuntu - as I shrink the page, the left hand search box
    cuts off the three dots and then the whole pane is enlarged once
    we've started to clip the 'e' in 'site', taking more space from the
    main pane.
  * Chrome on Ubuntu - the same general behavior as Firefox except it
    clips 'site' and once it gets to the point it would start clipping
    'this' the pane is expanded, and again, the middle pane is reduced
    to make up for the difference.
  * Firefox on Windows - same behavior as on Ubuntu.
  * Chrome on Windows - same behavior as on Ubuntu
  * Firefox on Mac - same behavior as Ubuntu
  * Chrome on Mac - same behavior as Ubuntu
  * Safari on Mac - Default window size comes up with the 'e' clipped
    off on 'site'.  Continues to clip like Chrome.  When I expand the
    window from the default size it shows 'site...' as expected.

Unless you have something completely different going on, I think this is 
working as expected.

-- 
Bill Stumbo
***@***.***

**Comment by nbriggs** [comment]
Safari on macOS, version 16.5 - yeah, it’s a page size issue: full screen with width 1366, and it leaves a big gap in the search text box with no indication that it has truncated.  If the window is just a little smaller than full width it cuts of exactly at “sit”, at full screen (where the fact that it’s cut off is obvious) it looks like this:



If I zoom out it looks like:



The display in the text entry box is a little weird when you type something longer than the size of the box, too, and that’s the same on Safari on the mac and Firefox on Ubuntu.  Normal interaction on the Mac is that you don’t ever display text over the magnifying glass icon.

— Nick

**Pull request merged**